hackathon_site/dashboard/frontend/src/components/dashboard/TeamCard/TeamCard.js

```diff
@@ -0,0 +1,64 @@
+import React from "react";
+import Paper from "@material-ui/core/Paper";
+import Container from "@material-ui/core/Container";
+import Typography from "@material-ui/core/Typography";
+import LinkIcon from "@material-ui/icons/Link";
+import MapIcon from "@material-ui/icons/Map";
+import GetAppIcon from "@material-ui/icons/GetApp";
+import styles from "./TeamCard.module.scss";
+
+function handleClick() {
+    alert("You clicked the link");
+}
+
+function ItemIcon(props) {
+    if (props.icon === "location") {
+        return <MapIcon onClick={handleClick}></MapIcon>;
+    } else if (props.icon === "link") {
+        return (
+            <LinkIcon href="https://css-tricks.com/fixing-tables-long-strings/"></LinkIcon>
+        );
+    } else if (props.icon === "download") {
+        return <GetAppIcon onClick={handleClick}></GetAppIcon>;
+    }
+    return;
+}
+
+function CardItem(props) {
+    return (
+        <tr>
+            <td className={styles.name}>
+                <Typography>{props.name}</Typography>
+            </td>
+            <td>
+                <ItemIcon url={props.url} icon={props.icon} />
+            </td>
+        </tr>
+    );
+}
+
+function CardTable(props) {
```

Comment:
In addition to replacing the table, just put all of this code inline with the component below. No need to have it as a separate component.